lib/script-loader.php

```diff
@@ -63,6 +92,7 @@ function gutenberg_enqueue_global_styles() {
  * @since 6.2.0
  */
 function gutenberg_enqueue_global_styles_custom_css() {
+	_deprecated_function( __FUNCTION__, 'Gutenberg 17.8.0', 'gutenberg_enqueue_global_styles' );
```

Comment:
I presume we also need to add a `@deprecated 6.6 Use wp_enqueue_global_styles instead` PHP doc, or something along those lines.